buildspec.yml

```diff
@@ -0,0 +1,35 @@
+version: 0.2
+
+env:
+  variables:
+    SERVICE_NAME: sbl-filing-api
+    CONTACTS_SECRET: cfpb/team/regtech/contact-info
+    IMAGE_SCANNER_SECRET: cfpb/team/regtech/image-scanner-creds
+    SMTP_CREDS_SECRET: cfpb/team/regtech/smtp-ses-creds
+  secrets-manager:
+    EMAIL_TO: "${CONTACTS_SECRET}:developers_all"
+    IMAGE_SCANNER_URL: "${IMAGE_SCANNER_SECRET}:url"
+    IMAGE_SCANNER_USERNAME: "${IMAGE_SCANNER_SECRET}:username"
+    IMAGE_SCANNER_PASSWORD: "${IMAGE_SCANNER_SECRET}:password"
+    SMTP_HOST: "${SMTP_CREDS_SECRET}:mail_server"
+    SMTP_PORT: "${SMTP_CREDS_SECRET}:smtp_port"
+    SMTP_USERNAME: "${SMTP_CREDS_SECRET}:username"
+    SMTP_PASSWORD: "${SMTP_CREDS_SECRET}:password"
+
```

Comment:
Is the SMTP stuff there to mail codebuild reports?

Reply:
Yessir. It currently send out an email when image scans fail, but we could expand it to do more if it's useful.